File: VB.NET PROJELER/Arduino Led Brightness/Kaynak kod/Arduino Led Brightness/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using System.IO.Ports;

namespace Arduino_Led_Brightness
{
    public partial class Form1 : Form
    {
        SerialPort serialport;
        public Form1()
        {
            InitializeComponent();
            serialport = new SerialPort();
            serialport.BaudRate = 9600;
           }
        private void Form1_Load(object sender, EventArgs e)
        {
            textBox1.Text = "COM4";
            groupBox2.Enabled = false;
            Button2.Enabled = false;
        }

        //Timer_Tick
         //Durdur
            private void label3_Click(object sender, EventArgs e)
        {
            System.Diagnostics.Process.Start("www.facebook.com/liveenes");
        }

        private void Button1_Click(object sender, EventArgs e)
        {
                            timer1.Start();
                try
                {
                                 serialport.PortName = textBox1.Text;
                    if (!serialport.IsOpen)
                        serialport.Open();
                    MessageBox.Show("Bağlandi");
                    groupBox2.Enabled = true;
                    Button2.Enabled = true;
                    Button1.Enabled = false;
                    textBox1.Enabled = false;
                }
                catch
                {
                    MessageBox.Show("Seri Port Hatası!");
                    Button1.Enabled = true;
                    Button2.Enabled = false;
                }
                    }

        private void Button2_Click(object sender, EventArgs e)
        {
                            timer1.Stop();
                serialport.Close();
                groupBox2.Enabled = false;
                Button2.Enabled = false;
                Button1.Enabled = true;
                textBox1.Enabled = true;
        }

        private void timer1_Tick(object sender, EventArgs e)
        {
            try
            {
                string ledval = trackBar1.Value.ToString();
                textBox2.Text = ledval;
                serialport.WriteLine(ledval);
                serialport.WriteLine(",");
            }
            catch (Exception ex) { }
        }
                    private void timer2_Tick(object sender, EventArgs e)
        {
            label3.Text = label3.Text.Substring(1) + label3.Text.Substring(1, 0) + label3.Text.Substring(0, 1);
        }
    }
}

File: VB.NET PROJELER/METİN2 TORRENTSİZ GİRİŞ/METİN2 TORRENTSİZ GİRİŞ/Form1.vb
Public Class Form1

    
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Timer1.start()


        Shell("Metin2Client.bin")
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        Timer2.Start()
        Shell("Metin2")
    End Sub

    Private Sub Timer1_Tick(sender As Object, e As EventArgs) Handles Timer1.Tick
        If ProgressBar1.Value = ProgressBar1.Maximum Then
            ProgressBar1.Value = 1
            Timer1.Stop()

        Else
            ProgressBar1.Value = ProgressBar1.Value + 1

        End If
    End Sub

    Private Sub Timer2_Tick(sender As Object, e As EventArgs) Handles Timer2.Tick
        If ProgressBar2.Value = ProgressBar2.Maximum Then
            ProgressBar2.Value = 1
            Timer1.Stop()
            MsgBox("METİN2 TORRENT GÜNCELLEME BAŞLATILIYOR...")

        Else
            ProgressBar2.Value = ProgressBar2.Value + 1

        End If
    End Sub
End Class
